odalic/src/main/java/cz/cuni/mff/xrg/odalic/input/CsvConfiguration.java

```diff
@@ -127,7 +127,7 @@ public Character getCommentMarker() {
   }
 
 
-  CSVFormat toApacheConfiguration() {
+  public CSVFormat toApacheConfiguration() {
```

Comment:
OK for now. But I will push for moving this method to a separate class in the future as it breaks the separation of concerns rule.